api/server/sdk/bucket_ops_test.go

```diff
@@ -0,0 +1,86 @@
+/*
+Package sdk is the gRPC implementation of the SDK gRPC server
+Copyright 2019 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package sdk
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/libopenstorage/openstorage/api"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBucketCreate(t *testing.T) {
```

Comment:
Can we add two more tests `TestBucketCreateFail` and `TestBucketDeleteFail`? You can inject an error to be returned with the following block:
```
	// Create CreateBucket response
	s.MockBucketDriver().
		EXPECT().
		CreateBucket(name).
		Return("", errors.New("failed")).
		Times(1)
```

and similarly for Delete

Reply:
Added TestBucketCreateFailure, TestBucketDeleteFailure. 
Also updated the previous test names to include "Success".